### ISSUE TITLE
 [Word Mobile]: NVDA is not announcing the proofing issues in the word universal.

### ISSUE THREAD

**Issue by WiproPSAccTest** [nvaccess/nvda#8281]
### Steps to reproduce:
1.     Open Word doc in the Universal app.
2.     Type some misspelled word and press space.
3.     Observe that NVDA is not announcing the word as misspelled.
4.     Now navigate throughout the doc having grammatical errors and Advanced proofing errors.
5.      Observe that NVDA is not announcing any of these.
 

### Expected behavior:
NVDA should announce all the available different types of errors in the doc.

### Actual behavior:
NVDA is not announcing any of the proofing error in the doc.

### System configuration:
NVDA version: 2018.1.1
App Name: Word
NVDA Installed or portable: Installed
**Other information:**
Windows version:
> Example: Windows 10 Version 1803 Build 17134.48
Name and version of other software in use when reproducing the issue:

### Other questions:
Does the issue still occur after restarting your PC?
Yes
Have you tried any other versions of NVDA?
No
> Please list them and the result

**Comment by WiproPSAccTest** [comment]
![misspelled word](https://user-images.githubusercontent.com/22956130/40101729-f5770b76-5905-11e8-85b1-e425b64bfce8.png)

**Comment by ehollig** [comment]
I assume this was accidentally closed.

**Comment by Adriani90** [comment]
I cannot reproduce this issue. Enabling the coresponding reporting option in NVDA's document formatting settings works as expected in Word. Closing as works for me with NVDA 2020.1. Please comment on this if you are still having this issue and we can reopen. Thanks.
